app/controllers/concerns/idv/doc_auth_vendor_concern.rb

```diff
@@ -50,7 +50,7 @@ def socure_user_set
 
     def add_user_to_socure_set
       uuid = current_user&.uuid
-      if uuid.nil? && is_defined?(document_capture_user)
+      if uuid.nil? && defined?(document_capture_user)
```

Comment:
this should have test coverage since it's a follow-up to #11827 that lacked test coverage